### PR TITLE
SSC-637: Introduced description in README file

### PR DESCRIPTION
**Pull request by sprymiker**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,158 @@
...
+[![microbadger image](https://images.microbadger.com/badges/image/spryker/php.svg)](https://microbadger.com/images/spryker/php "Get your own image badge on microbadger.com")
+
+# Description
+
 Extends official PHP Docker images with extensions and tools to be able to run Spryker on.
+
+* Based on: `Alpine 3.9`
+* Users: `root`, `spryker`
+* Working directory: `/data`
+* Includes:
```
Comment (gvoit) [inline README.md]: What about DBs? Do we need to include them as well?

File: README.md [hunk 2]
```diff
@@ -1,2 +1,158 @@
...
+[![Docker Stars](https://img.shields.io/docker/stars/spryker/php.svg)](https://store.docker.com/community/images/spryker/php)
+[![Docker Pulls](https://img.shields.io/docker/pulls/spryker/php.svg)](https://store.docker.com/community/images/spryker/php)
+[![microbadger version](https://images.microbadger.com/badges/version/spryker/php.svg)](https://microbadger.com/images/spryker/php "Get your own version badge on microbadger.com")
+[![microbadger image](https://images.microbadger.com/badges/image/spryker/php.svg)](https://microbadger.com/images/spryker/php "Get your own image badge on microbadger.com")
+
+# Description
+
 Extends official PHP Docker images with extensions and tools to be able to run Spryker on.
+
+* Based on: `Alpine 3.9`
```
Comment (konstantin-tishkin-spryker) [inline README.md]: Technically our image is based on official PHP (which is based on Alpine 3.9).
Maybe we should mention that?

File: README.md [hunk 3]
```diff
@@ -1,2 +1,158 @@
...
+  * `hirak/prestissimo`
+  * `aws/aws-sdk-php`
+
+> Note: Provided images require additional configuration for development, staging and production use.
+
+## Tags
+
+| Tag     | PHP version     | Details     | Dockerfile     |
+| :------------- | :------------- | :------------- | :------------- |
+| [spryker/php:latest](https://hub.docker.com/r/spryker/php/tags) | 7.2.16 | [![](https://images.microbadger.com/badges/image/spryker/php:latest.svg)](https://microbadger.com/images/spryker/php "Get your own image badge on microbadger.com") | [:link:](https://github.com/spryker/docker-php/blob/master/7.2/Dockerfile) |
```
Comment (gvoit) [inline README.md]: Isn't `spryker/php:latest` `7.3.3`, is it?

File: README.md [hunk 4]
```diff
@@ -1,2 +1,158 @@
...
+| [spryker/php:7.3](https://hub.docker.com/r/spryker/php/tags)  | 7.3.3 | [![](https://images.microbadger.com/badges/image/spryker/php:7.3.svg)](https://microbadger.com/images/spryker/php "Get your own image badge on microbadger.com") | [:link:](https://github.com/spryker/docker-php/blob/master/7.3/Dockerfile) |
+| [spryker/php:7.2](https://hub.docker.com/r/spryker/php/tags)  | 7.2.16 | [![](https://images.microbadger.com/badges/image/spryker/php:7.2.svg)](https://microbadger.com/images/spryker/php "Get your own image badge on microbadger.com") | [:link:](https://github.com/spryker/docker-php/blob/master/7.2/Dockerfile) |
+| [spryker/php:7.1](https://hub.docker.com/r/spryker/php/tags)  | 7.1.27 | [![](https://images.microbadger.com/badges/image/spryker/php:7.1.svg)](https://microbadger.com/images/spryker/php "Get your own image badge on microbadger.com") | [:link:](https://github.com/spryker/docker-php/blob/master/7.1/Dockerfile) |
+
+
+## How to use
+
+### Pull image
+```bash
+$ docker pull spryker/php
```
Comment (gvoit) [inline README.md]: What is the reason to provide these two lines in sample? Would be enough to have only one of them?

File: README.md (hunk 3 above)
Comment (sprymiker) [reply README.md]: No. 7.2

File: README.md (hunk 4 above)
Comment (sprymiker) [reply README.md]: I want to pull both images

File: README.md (hunk 2 above)
Comment (sprymiker) [reply README.md]: The phrase above `Extends official PHP Docker images with extensions and tools to be able to run Spryker on.` already describes it.

File: README.md [hunk 5]
```diff
@@ -1,2 +1,163 @@
...
+  * MySQL client
+  * CURL
+  * OpenSSH
+  * Composer
+  * `hirak/prestissimo`
+  * `aws/aws-sdk-php`
+
+> Note: Provided images require additional configuration for development, staging and production use.
+
+## Tags
```
Comment (marek-obuchowicz) [inline README.md]: Are you sure you want to include list of tags here? That's a manual process to update this file each time new PHP version is released

Comment (sprymiker) [reply README.md]: Yep. We will do that in Dockerfiles and in README.

**Pull request merged**